subsys/nrf_security/Kconfig.psa

```
@@ -104,6 +104,20 @@ config PSA_ITS_ENCRYPTED
 	help
 	  Enables authenticated encryption for PSA Internal Trusted Storage files
 
+config PSA_CRYPTO_SYS_INIT
+	bool
+	prompt "Enable a Zephyr SYS_INIT that invokes psa_crypto_init"
```

```suggestion
	bool "Enable a Zephyr SYS_INIT that invokes psa_crypto_init"
```
also fix prompt as per https://docs.zephyrproject.org/latest/build/kconfig/tips.html#prompt-strings

---

fixed